packages/itwin/map-layers/src/ui/widget/FeatureInfoWidget.tsx

```diff
@@ -58,6 +61,20 @@ export function MapFeatureInfoWidget({ featureInfoOpts }: MapFeatureInfoWidgetPr
     setSize({ width: w, height: h });
   }, []);
 
+  const copyButton = React.useCallback(
+    (props: ActionButtonRendererProps) => props.isPropertyHovered &&
+      <div>
+        <IconButton styleType="borderless" onClick={()=>{
+          const value = props.property.value;
+          if (value !== undefined && value.hasOwnProperty("displayValue") )
+            navigator.clipboard.writeText((value as PrimitiveValue).displayValue??"").catch((_) => { });
```

Comment:
Does this call need a `.catch()` if its empty?

Reply:
> Does this call need a `.catch()` if its empty?

writetext is async, and wanted to avoid lint errors about no-floating-promises